### PR TITLE
Adding article for deleting SSO user

### PR DESCRIPTION
**Pull request by suehighmoor**

### CODE REVIEW

File: articles/other/other-how-delete-sso-user.md [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+toc_sub1: 
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: Delete SSO users
+toc_fullpath: How To/other-how-delete-sso-user.md
+toc_mdlink: other-how-delete-sso-user.md
+---
+
+# How to delete single sign on users
```
Comment (boardfish) [inline articles/other/other-how-delete-sso-user.md]: I'd prefer use of 'Single Sign-On', both here and more widely across the site wherever possible.

File: articles/other/other-how-delete-sso-user.md [hunk 2]
```diff
@@ -0,0 +1,190 @@
+---
+title: How to delete single sign on users | UKCloud Ltd
```
Comment (boardfish) [inline articles/other/other-how-delete-sso-user.md]: Perhaps 'delete' could be better worded as 'remove access from' – IDAM users don't have the necessary access to be able to delete a user from the system entirely, but they can remove the user's access to services and accounts through the process outlined in the article.

File: articles/other/other-how-delete-sso-user.md [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+toc_title: Delete SSO users
+toc_fullpath: How To/other-how-delete-sso-user.md
+toc_mdlink: other-how-delete-sso-user.md
+---
+
+# How to delete single sign on users
+
+## Overview
+
+When you use the UKCloud Portal to remove a user from an account that includes single sign on (SSO) enabled services, it's important to note that this does not remove the corresponding SSO user. To remove the SSO user, you must use the Identity & Access Management (IDAM) API.
```
Comment (boardfish) [inline articles/other/other-how-delete-sso-user.md]: It might be better to add the abbreviation to the article title and rephrase this as 'SSO-enabled services' as it is on the next line. The lack of a hyphen before 'enabled' is inconsistent.

File: articles/other/other-how-delete-sso-user.md [hunk 4]
```diff
@@ -0,0 +1,190 @@
...
+     "session_state": "342ab476-d286-41b9-8b3a-1a236cceefc1"
+    }
+    ```
+
+    For convenience, the token is saved to the `token` variable in your current shell session. Use this variable in the `Authorization: Bearer` header for all further calls to the IDAM API.
+
+2. List users and note the ID of the user you want to delete:
+
+    ``` bash
+    curl -s -H "Authorization: Bearer $token" https://idam.ukcloud.com/v1/users | jq -r '.[] | .id + " " + .username'
```
Comment (boardfish) [inline articles/other/other-how-delete-sso-user.md]: The `jq` filter on this request returns the list of users in the following format:

```
0744f665-2d7a-483c-adb3-72b0effe2356 acc1admin
1d9e52e6-f833-47b9-9cfa-be727510c910 acc1usr1
a089698b-8f32-4a32-ab71-790b015aa662 acc1usr2
```

This should replace the code block on line 69 more cleanly.

File: articles/other/other-how-delete-sso-user.md [hunk 5]
```diff
@@ -0,0 +1,190 @@
...
+          "view": true,
+          "mapRoles": true,
+          "impersonate": false,
+          "manage": true
+        }
+      },
+    ]
+    ```
+
+3. Find the user that you want to delete and make a note of that user's ID. For example, to delete the user `acc1usr2`, you would need the ID `a089698b-8f32-4a32-ab71-790b015aa662`.
```
Comment (boardfish) [inline articles/other/other-how-delete-sso-user.md]: Perhaps it could be suggested to keep this in a variable for reuse if the user should be removed from multiple groups, e.g.

```
acc1usr2=a089698b-8f32-4a32-ab71-790b015aa662
curl -X DELETE -H "Authorization: Bearer $token" https://idam.ukcloud.com/v1/users/$acc1usr2/groups/008d3512-54f9-439e-800a-7b0b5ef76438
```

File: articles/other/other-how-delete-sso-user.md [hunk 6]
```diff
@@ -0,0 +1,190 @@
...
+
+4. List all the groups that you have permission to manage:
+
+    ``` bash
+    curl -s -H "Authorization: Bearer $token" https://idam.ukcloud.com/v1/groups | jq -r '.[].subGroups[] | .id + " " + .name'
```
Comment (boardfish) [inline articles/other/other-how-delete-sso-user.md]: Again, the `jq` filter here returns each top-level group's subgroups in the format:

```
008d3512-54f9-439e-800a-7b0b5ef76438 idam:Account_101#User
a4ebc856-222f-4686-a4bf-239c21ffa010 idam:Account_101#Admin
```

File: articles/other/other-how-delete-sso-user.md (hunk 2 above)
Comment (suehighmoor) [reply articles/other/other-how-delete-sso-user.md]: Fixed

File: articles/other/other-how-delete-sso-user.md (hunk 1 above)
Comment (suehighmoor) [reply articles/other/other-how-delete-sso-user.md]: Fixed

File: articles/other/other-how-delete-sso-user.md (hunk 3 above)
Comment (suehighmoor) [reply articles/other/other-how-delete-sso-user.md]: Fixed

File: articles/other/other-how-delete-sso-user.md (hunk 4 above)
Comment (suehighmoor) [reply articles/other/other-how-delete-sso-user.md]: Fixed

File: articles/other/other-how-delete-sso-user.md (hunk 5 above)
Comment (suehighmoor) [reply articles/other/other-how-delete-sso-user.md]: Fixed

File: articles/other/other-how-delete-sso-user.md (hunk 6 above)
Comment (suehighmoor) [reply articles/other/other-how-delete-sso-user.md]: Fixed

File: articles/other/other-how-delete-sso-user.md (hunk 2 above)
Comment (boardfish) [reply articles/other/other-how-delete-sso-user.md]: The latter half of the document still refers to this as deleting an SSO user. Also, it's dawned on  me that 'removing access **rights**' sounds a bit better now.
